linera-service/src/lib.rs

```diff
@@ -8,6 +8,7 @@ pub mod chain_listener;
 pub mod client;
 pub mod config;
 pub mod grpc_proxy;
+pub mod kubernetes;
```

Comment:
Add `#[cfg(feature = "kube")]` before this line here to make the module feature-gated.